DirectEmployers/MyJobs
# [PD-3327] suggested roles

## What's this PR do?
Adds Suggested Roles to user management
## Screenshots (if appropriate)
<img width="979" alt="screen shot 2017-11-27 at 4 24 38 pm" src="https://user-images.githubusercontent.com/26224950/33290067-87cabee8-d38f-11e7-8357-19d27e5e2c7c.png">

## Where should the reviewer start?
FrontEnd: A new file SuggestedRoles.jsx has been added
BackEnd: 2 new functions for suggested roles have been added in myjobs/views.py
## How should this be manually tested?
Under Manage Users, check for a new tab Suggested Roles
A Suggested Role can be added in admin.
Click through buttons "Save Role"/"Reject"
## Any background context you want to provide?

## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
